lib/usecases/oath/login_use_case.dart

```diff
@@ -19,9 +24,23 @@ class LoginUseCase extends UseCase<OAuthLogin, LoginParams> {
         email: params.email,
         password: params.password,
       );
+      _storeTokens(result);
       return Success(result);
     } catch (exception) {
       return Failed(UseCaseException(exception));
     }
   }
+
+  Future<Result<void>> _storeTokens(OAuthLogin oauthLogin) async {
+    try {
+      await _secureStorage.storeId(oauthLogin.id);
+      await _secureStorage.storeTokenType(oauthLogin.tokenType);
+      await _secureStorage.storeAccessToken(oauthLogin.accessToken);
+      await _secureStorage.storeExpiresIn('${oauthLogin.expiresIn}');
+      await _secureStorage.storeRefreshToken(oauthLogin.refreshToken);
+      return Success(null);
+    } catch (exception) {
+      return Failed(UseCaseException(exception));
```

Comment:
@nmint8m I think we can ignore the failure if we're failed to store the token locally. We just need to let user continue using the app after login. If the token is failed to save, we will force them to login again later (maybe opening the app next time). What do you think?

Reply:
I don't think so. Because we must use the token while using the app by attaching the token to the API header for authorization. So if we failed the step to store new tokens, the token in API header is invalid, and we cannot call API. 🤔

Reply:
make sense. So let's keep it as it is 👌